chia/plotting/manager.py

```diff
@@ -205,43 +263,52 @@ def process_file(file_path: Path) -> Dict:
                         )
                         return new_provers
 
-                    (
-                        pool_public_key_or_puzzle_hash,
-                        farmer_public_key,
-                        local_master_sk,
-                    ) = parse_plot_info(prover.get_memo())
-
-                    # Only use plots that correct keys associated with them
-                    if self.farmer_public_keys is not None and farmer_public_key not in self.farmer_public_keys:
-                        log.warning(f"Plot {file_path} has a farmer public key that is not in the farmer's pk list.")
-                        self.no_key_filenames.add(file_path)
-                        if not self.open_no_key_filenames:
-                            return new_provers
+                    cache_entry = self.cache.get(prover.get_id())
+                    if cache_entry is None:
+                        (
+                            pool_public_key_or_puzzle_hash,
+                            farmer_public_key,
+                            local_master_sk,
+                        ) = parse_plot_info(prover.get_memo())
 
-                    if isinstance(pool_public_key_or_puzzle_hash, G1Element):
-                        pool_public_key = pool_public_key_or_puzzle_hash
-                        pool_contract_puzzle_hash = None
-                    else:
-                        assert isinstance(pool_public_key_or_puzzle_hash, bytes32)
-                        pool_public_key = None
-                        pool_contract_puzzle_hash = pool_public_key_or_puzzle_hash
-
-                    if (
-                        self.pool_public_keys is not None
-                        and pool_public_key is not None
-                        and pool_public_key not in self.pool_public_keys
-                    ):
-                        log.warning(f"Plot {file_path} has a pool public key that is not in the farmer's pool pk list.")
-                        self.no_key_filenames.add(file_path)
-                        if not self.open_no_key_filenames:
-                            return new_provers
+                        # Only use plots that correct keys associated with them
```

Comment:
`s/that correct/that have correct/`

Reply:
Unrelated to this PR, its just changed indentation here.